### ISSUE TITLE
CzechVR, LethalHardcoreVR, and VRHush scrappers not working.

### ISSUE THREAD

**Issue by dudeGuy2342** [xbapps/xbvr#1718]
CzechVR doesn't scrape dates beyond first page.
LethalHardcoreVR scrapes nothing.
VRHush VRP Hub scrapes scenes from first page twice and none beyond.
VRHush doesn't scrape scenes beyond first page.

**Comment by theRealKLH** [comment]
Using an empty DB... 
* CzechVR, I just scraped 1372 scenes. You may be a few releases behind. there was a czechvr scraper update last month
* VRHush/VRAllure, I just scraped 809 (check that you aren't getting caught by captcha)
* VRHush VRP Hub, I just scraped 551 (w/dupes). Scraper needs an update (or just don't use it)
* LethalHardcore recently updated their site to use React ( #1696 ), so that scraper is DEAD! you'll have to do "custom scraper" (Advanced / Create Custom Site) via SLR or POVR or VRPORN or VRPHUB

**Comment by dudeGuy2342** [comment]
Nope, I was using the current version. When I roll back to 0.4.22 I am able to scrape CzechVR with no dates past the first page, and am able to fully scrape. 0.4.25 have the states issues.

**Comment by Saturnix** [comment]
> Nope, I was using the current version. When I roll back to 0.4.22 I am able to scrape all CzechVR scenes, just no dates past the first page, and am able to fully scrape VRHush. However, 0.4.25 have the stated issues.

When scraping single scenes, use links from czechvrnetwork.com instead of czechvr.com

The bug only appears on the latter. I think it used to work on both, but I'm unsure.

**Comment by SturmB** [comment]
> LethalHardcore recently updated their site to use React ( https://github.com/xbapps/xbvr/issues/1696 ), so that scraper is DEAD! you'll have to do "custom scraper" (Advanced / Create Custom Site) via SLR or POVR or VRPORN or VRPHUB

@theRealKLH Could you explain the best procedure for doing this? I've attempted to add a custom site for LethalHardcore using SLR and although I'm pretty sure I filled the fields in correctly, it only seems to scrape one scene.

![image](https://github.com/user-attachments/assets/085e8047-2edc-483b-be59-1a8ea6eb82e5)

**Comment by SturmB** [comment]
Please disregard my previous comment. I
